package.json

```diff
@@ -17,7 +17,8 @@
     "megs"
   ],
   "scripts": {
-    "pretest": "jshint --config ./node_modules/yui-lint/jshint.json ./index.js",
+    "lint": "eslint --fix index.js",
```

Comment:
Can you try `eslint --fix .`? That will ensure that even the tests are linted too..